src/xnatdcat/xnat_parser.py

```diff
@@ -59,10 +60,30 @@ def xnat_to_DCATDataset(project) -> DCATDataSet:
     return project_dataset
 
 
-def XNAT_to_DCAT(session) -> Graph:
-    """Creates a DCAT-AP compliant Catalog of Datasets from XNAT
+def xnat_to_catalog(session, config) -> DCATCatalog:
```

Comment:
Inconsistent naming convention compared to `xnat_to_DCATDataset`.

Reply:
Absolutely right. I renamed the function to xnat_to_DCATCatalog. For consistency I renamed xnat_to_DCAT to xnat_to_RDF